analytic_hours_block/hours_block.py

```diff
@@ -173,6 +177,16 @@ def _get_invoice(self, cr, uid, ids, context=None):
                               in invoice.account_hours_block_ids])
         return list(block_ids)
 
+    def _get_invoice_line(self, cr, uid, ids, context=None):
+        invoice_ids = set()
+        line_obj = self.pool.get('account.invoice.line')
+        block_obj = self.pool.get('account.hours.block')
```

Comment:
This form is better: `block_obj = self.pool['account.hours.block']`.
Explanation:
When `account.hours.block` is not in `self.pool` for some reason:

* `self.pool.get('account.hours.block')` will return None, so when later you do `block_obj._get_invoice(...)`, will raise an AttributeError `'NoneType' object has no attribute '_get_invoice'` which is hard to debug
* `self.pool['account.hours.block']` will fail directly, which makes it easier to find the issue